pyuvdata/utils.py

```diff
@@ -34,6 +34,27 @@ def _str_to_bytes(s):
     def _bytes_to_str(b):
         return b.decode('utf8')
 
+# polarization constants
+# XXX is the 'p' capitalized or not?
+POLSTR_LIST = ['YX', 'XY', 'YY', 'XX', 'LR', 'RL', 'LL', 'RR', '', 'pI', 'pQ', 'pU', 'pV']
+POLSTR_DICT = {'PI': 1, 'PQ': 2, 'PU': 3, 'PV': 4,
+               'I': 1, 'Q': 2, 'U': 3, 'V': 4,
+               'RR': -1, 'LL': -2, 'RL': -3, 'LR': -4,
+               'XX': -5, 'YY': -6, 'XY': -7, 'YX': -8}
+
+CPOL_DICT = {'XX': 'XX', 'YY': 'YY', 'XY': 'YX', 'YX': 'XY',
+             'JXX': 'jxx', 'JYY': 'jyy', 'JXY': 'jyx', 'JYX': 'jxy',
+             'RR': 'RR', 'LL': 'LL', 'RL': 'LR', 'LR': 'RL',
+             'JRR': 'jrr', 'JLL': 'jll', 'JRL': 'jlr', 'JLR': 'jrl',
+             'I': 'I', 'Q': 'Q', 'U': 'U', 'V': 'V',
+             'PI': 'pI', 'PQ': 'pQ', 'PU': 'pU', 'PV': 'pV'}
```

Comment:
I wonder whether we really need to have all these duplicates based on case. I think we should be able to fix the functions to deal with case properly.